linkedList.js

```diff
@@ -4,5 +4,110 @@
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
 function linkedListGenerator(){
+  var head = null;
+  var tail = null;
+  var node = function() {
+    this.value = null;
+    this.next = null;
+  };
 
+ function getHead(){
+  return head;
+ }
+ function getTail(){
+  return tail;
+ }
+
+ function add(item){
+  var insert = new node();
+  insert.value = item;
+    if(head === null){
+      head = insert;
+      head.next = tail;
+      tail = insert;
+    }
+    else{
+      tail.next = insert;
+      tail = insert;
+    }
+    return insert;
+ }
+
+  function get(num){
```

Comment:
I know the tests don't check for it, but what happens if `num` was a negative number? I think it should also return `false`.